### PR TITLE
Previous translations showing up fix

### PR DESCRIPTION
**Pull request by JorritO**

### CODE REVIEW

File: Zeeguu Reader/Zeeguu Reader/Utils.swift [hunk 1]
```diff
@@ -42,25 +42,49 @@ class Utils {
...
+		let contextStr: NSMutableString = NSMutableString(string: context.string)
+		
+		// context is the context that may contain other translations
+		
+		for var i = context.length - 1; i >= 0; --i {
+			let effectiveRange = NSRangePointer()
+			let color = context.attribute(NSForegroundColorAttributeName, atIndex: i, effectiveRange: effectiveRange)
+			
+			if let c = color {
+				if c.isEqual(UIColor.lightGrayColor()) {
```
Comment (mircealungu) [inline Zeeguu Reader/Zeeguu Reader/Utils.swift]: Hardcoding lightGrayColor here will break the code in the future if we ever decide to use another color. 
As you know, everything in CS can be solved by another level of indirection... ;)

File: Zeeguu Reader/Zeeguu Reader/Utils.swift [hunk 2]
```diff
@@ -42,25 +42,49 @@ class Utils {
...
+					if effectiveRange != nil {
+						let r = effectiveRange.move()
+						contextStr.replaceCharactersInRange(r, withString: "")
+						i = r.location + 1
+					} else {
+						contextStr.replaceCharactersInRange(NSMakeRange(i, 1), withString: "")
```
Comment (mircealungu) [inline Zeeguu Reader/Zeeguu Reader/Utils.swift:73]: can this branch ever be taken? 
i don't think you can ever end up with a single character being highlighted. 
or am i missing something?

File: Zeeguu Reader/Zeeguu Reader/Utils.swift (hunk 1 above)
Comment (JorritO) [inline Zeeguu Reader/Zeeguu Reader/Utils.swift]: Added central class for getting colors (called `AppColor`)

File: Zeeguu Reader/Zeeguu Reader/Utils.swift (hunk 2 above)
Comment (JorritO) [inline Zeeguu Reader/Zeeguu Reader/Utils.swift:73]: Probably not, but its just there 'just in case'

**Pull request merged**